js/index.js

```diff
@@ -40,3 +40,64 @@ const siteContent = {
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+let headerImage = document.getElementById("cta-img");
+headerImage.setAttribute('src',siteContent["cta"]["img-src"] )
+
+let middleImage = document.getElementById("middle-img");
+middleImage.setAttribute('src', "img/mid-page-accent.jpg")
+
+let navLinks = document.getElementsByTagName('a');
```

Comment:
While you can use getElementsByTagName if you use querySelectorAll it opens up a few array methods right out of the box. Also if you were to use ArrayFrom you get access to ALL of the array methods. A for loop is definitely something you can still use, but being able to easily map or forEach is super beneficial